src/porepy/models/momentum_balance.py

```diff
@@ -30,6 +30,13 @@ class that defines the variables and discretization.
 class MomentumBalanceEquations(pp.BalanceEquation):
     """Class for momentum balance equations and fracture deformation equations."""
 
+    solid: pp.SolidConstants
+    """Solid constant object that takes care of scaling of solid-related quantities.
+    Normally, this is set by a mixin of instance
```

Comment:
Is this being used?

Reply:
Since the tolerance is now in SolidConstants i put this in to avoid the mypy test complaining